setup.py

```diff
@@ -103,10 +115,11 @@ def run(self):
       license='MIT License',
       platforms=['POSIX'],
       package_dir=package_dir,
-      py_modules=['typing'] if sys.version_info < (3, 5, 0) else [],
```

Comment:
OK, that explains it. I'm just used to `py_modules` being defined on the packages that *implement* a feature, not for dependencies.